source/server/spongineer/incompatible.rst

```diff
@@ -52,6 +52,25 @@ Hammer Core
 
 - Solution: Change ``<entry key="World.ITickable.Override">true</entry>`` to ``false`` in the ``/asm/hammercore.xml`` file
 
+ICBM Classic
+~~~~~~~~~~~~
+- Versions: All
+- Problem: Missiles stop in midair and appear to lag in place.
+- Solution: Add the following mod specific configuration to the ``sponge/global.conf`` entity-activation-range-> mods section
+
+.. code-block:: none
+
+            icbmclassic {
+                defaults {
+                    misc=16
+                }
+                # If 'false', entity activation rules for this mod will be ignored and always tick.
```

Comment:
IMO instead of saying what is happening, we should say what the user should do:
`Set this to false to ...`, or `Copy the entire section into your config`, ...